mevblocker/web
# Update partner logos

# Summary

Adds more partner logos

Keystone
Swapr
StakeDAO
Safe
Oasis
Swarm
Onramper
ShapeShift
Gearbox
Gelato
VirtuSwap
Builder0x69
ZeroMev
BitKeep
Liminal
Castle
Ambire
Bloxroute
1inch

Might need to be added (pending)
Dodo

Logo doesn't look great
Builder0x69

# To test
- open demo
- scroll down to partners section
- click logos and check if links are correct

![Screenshot 2023-04-04 at 14 50 13](https://user-images.githubusercontent.com/34926005/229797722-9f70b466-5101-4a40-84a9-9a10fed71958.png)
